front/pages/w/[wId]/builder/assistants/index.tsx

```diff
@@ -216,15 +221,30 @@ export default function WorkspaceAssistants({
     }
   }, [tabScope]);
 
+  // Hack to display this page either from Builder or Chat Menu
+  const { conversations, isConversationsError } = useConversations({
```

Comment:
Please add a comment so it uses `ConversationLayout` once the build menu is gone 🙏

Reply:
Done!